code/modules/vtr13/connections/endorsements/endorsement_bishop.dm

```diff
@@ -0,0 +1,5 @@
+/datum/character_connection_type/endorsement/bishop
+	name = CONNECTION_ENDORSEMENT_BISHOP
+	required_faction = EXP_TYPE_LANCEA
+	desired_position = "Bishop"
+	minimum_endorser_rank = VAMP_RANK_NEONATE
```

Comment:
all roles other than carthian rep should have minimum_candidate_rank at ancilliae

Reply:
I will fix this